src/core/stream.c

```diff
@@ -574,6 +574,14 @@ QuicStreamShutdown(
         // and shutdown complete events now, if they haven't already been
         // delivered.
         //
+        if (Stream->Flags.RemoteCloseResetReliable || Stream->Flags.LocalCloseResetReliable) {
+             QuicTraceLogStreamWarning(
+                BadShutdownImmediate,
+                Stream,
+                "App tried ditching this stream but it still has stuff to do.");
+            QuicConnTransportError(Stream->Connection, QUIC_ERROR_STREAM_STATE_ERROR);
```

Comment:
Don't kill the whole connection because of this. Just log the warning and that's it.

Reply:
Sounds good.